lldb/test/API/lang/swift/async/tasks/info/TestSwiftTaskInfo.py

```diff
@@ -0,0 +1,27 @@
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import TestBase
+import lldbsuite.test.lldbutil as lldbutil
+
+import re
+
+
+def _tail(output):
+    """Delete the first line of output text."""
+    result, _ = re.subn(r"^.*\n", "", output, count=1)
+    return result
+
+
+class TestCase(TestBase):
+
+    @skipUnlessDarwin
```

Comment:
Is this conceptual for the features tested in `async/tasks`? Or is this a workaround for the Windows test failure linked above? In that case, can we mark it `xfail` instead, so that it pops up when this is fixed?

Reply:
this is specific to this `task info` command. A thread's current task is stored differently on Darwin, vs other platforms. This PR supports only Darwin, a follow up PR will need to support other platforms.